server-timing/.wp-env.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Do we need `.wp-env.json` in server-timing folder?

Reply:
@mukeshpanchal27 this was incorrectly copied by myself, I have since removed it.